File: week6/pset6/mario/more/mario.py
# Port of mario/more
from cs50 import get_int

height = get_int("Height: ")
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i + "  " + "#" * i)

File: week1/pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for pyramid height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Draw pyramid
    for (int i = 0; i < height; i++)
    {
        // Draw spaces
        for (int j = height - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        // Draw hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        // Draw spaces
        for (int j = 0; j < 2; j++)
        {
            printf(" ");
        }
        // Draw hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}